pkgs/development/tools/misc/universal-ctags/default.nix

```diff
@@ -21,6 +21,11 @@ stdenv.mkDerivation rec {
 
   configureFlags = [ "--enable-tmpdir=/tmp" ];
 
+  postPatch = ''
+    substituteInPlace main/options.c \
+      --replace "printf (\"  Compiled: %s, %s\n\", __DATE__, __TIME__);" ""
```

Comment:
What does this accomplish?

Reply:
It's to remove non-determinism, because those macros embed the compilation time in the binary. I added a comment

Reply:
Yuck :smile: We could advise upstream to not do that.